src/core/components/layout/stories/columns/collapse-below.tsx

```diff
@@ -28,9 +36,97 @@ export const collapseBelowTablet = () => (
 );
 
 collapseBelowTablet.story = {
-	name: 'collapse below tablet',
-	parameters: {
-		viewport: { defaultViewport: 'phablet' },
-		layout: 'fullscreen',
-	},
+	...defaultStoryOptions,
+	name: 'collapse below tablet with default spacing',
+};
+
+const collapseBelowTabletWithSpace = (space: ColumnsSpaceY) => (
+	<Container border={true}>
+		<Columns collapseBelow="tablet" spaceY={space}>
+			<Column>
+				<div css={contents}>1</div>
+			</Column>
+			<Column>
+				<div css={contents}>2</div>
+			</Column>
+			<Column>
+				<div css={contents}>3</div>
+			</Column>
+			<Column>
+				<div css={contents}>4</div>
+			</Column>
+		</Columns>
+	</Container>
+);
+
+export const collapseBelowTabletWithSpace1 = () =>
+	collapseBelowTabletWithSpace(1);
+
+collapseBelowTabletWithSpace1.story = {
+	...defaultStoryOptions,
+	name: 'collapse below tablet with space 1',
+};
```

Comment:
Not sure if there's a better way, with less copying-and-pasting. Also, this clutters up the stories for columns a bit. Any thoughts on how else this could work?

Reply:
I've tidied this up slightly so that the `no space` story follows the same pattern. Maybe this is fine as-is now that it's consistent?

Reply:
This is 👌  Thanks for taking the time to keep this tidy ❤️